Selenium_Basics/Selenium_Basics/UnitTest1.cs

```diff
@@ -0,0 +1,61 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenium_Basics
+{
+    public class EpamTests
+    {
+        private IWebDriver driver { get; set; }
+        private const string EpamHomePageUrl = "https://www.epam.com/";
+        private const string EpamHowWeDoItUrl = "https://www.epam.com/how-we-do-it";
+        private const string EpamOurWorkUrl = "https://www.epam.com/our-work";
+
+        [SetUp]
+        public void SetupBrowser()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Window.Maximize();
+        }
+
+
+        [Test]
+        public void TestEpamHomePage()
```

Comment:
We here check only navigation to landing page :) TestEpamHomePage -> CheckNavigationToEpamLandingPage